specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHub-preview.json

```
@@ -211,6 +261,92 @@
           }
         },
         "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "Event Hubs Cluster"
+        ],
+        "operationId": "Clusters_Delete",
+        "x-ms-examples": {
+          "ClusterDelete": {
+            "$ref": "./examples/Clusters/ClusterDelete.json"
+          }
+        },
+        "description": "Deletes an existing Event Hubs Cluster. This operation is idempotent.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/clusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Event Hubs Cluster successfully deleted."
+          },
+          "202": {
+            "description": "Event Hubs Cluster delete request accepted."
+          },
+          "204": {
+            "description": "No content."
+          },
+          "default": {
+            "description": "Event Hubs error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters/{clusterName}/namespaces": {
+      "post": {
+        "tags": [
+          "Event Hubs Namespace Enumeration in Cluster"
+        ],
+        "operationId": "Cluster_Namespace_List",
```

@FBettati 
Can you kindly fix this.
* Operation ID's first part should be plural. 'Operations' instead of 'Operation'
* Two underscores have no meaning, It would be better to just name it `Cluster_NamespaceList` or `Cluster_ListNamespace`
 * Putting two underscored will not give you two nested classes, we just generate the class for the what comes after the first underscore.

```suggestion
        "operationId": "Clusters_NamespaceList",
```